spec/book_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+require_relative '../teacher'
+require_relative '../student'
+
```

Comment:
Please add `bundle install` and `rspec` commands in README so a non-technical user of the project can easily set it up locally.

Reply:
Thank you. We have Updated the Readme file as you requested.